include/RAJA/policy/cuda.hpp

```diff
@@ -42,6 +42,7 @@
 #if defined(__NVCC__)
```

Comment:
Why is this guard needed here? The contents of the file are guarded by RAJA_ENABLE_CUDA and the other policy/cuda/*.hpp inclusions are not guarded. If the guard is actually needed, I think it should go inside the header.

Reply:
That was added by @willkill07 I think - nothing to do with this PR.